Gemfile

```diff
@@ -1,16 +1,16 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "3.2.2"
+ruby '3.2.2'
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.0.5"
+gem 'rails', '~> 7.0.5'
```

Comment:
- [ ] Please add the gem 'rack-corse' to avoid this issue 👇🏽 

```
Access to XMLHttpRequest at 'http://[::1]:3000/random_greeting' from origin 'http://localhost:3001' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```
You could see this [tutorial](https://youtu.be/sh4WrNGDvQM?t=58) about cors in rails.